pipelines/skylab/multiome/atac.wdl

```diff
@@ -30,6 +30,7 @@ workflow ATAC {
     Int num_threads_bwa = 128
     Int mem_size_bwa = 512
     String cpu_platform_bwa = "Intel Ice Lake"
+    String vm_size
```

Comment:
We should add this input to the ATAC Overview as well

Reply:
added, ty!